### PR TITLE
Fix typos in README

### PR DESCRIPTION
**Pull request by ndresx**
This fixes some typos in the documentation.

_`Yalc` is a really nice and very useful plugin, well done and thank you for creating something like this!_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,23 +88,23 @@ Some documented features might not has been published yet, see [change log](./CH
...
 - Replace it with published versions from remote repository when ready.
-- **NB!** - standard non-code files like `README`, `LICENCE` etc. will be included also, so you may want to excluded them in `.gitignore` with a line like `**/.yalc/**/*.md` or you may use `.yalcignore` not to include those files in package content.
+- **NB!** - standard non-code files like `README`, `LICENCE` etc. will be included also, so you may want to exclude them in `.gitignore` with a line like `**/.yalc/**/*.md` or you may use `.yalcignore` not to include those files in package content.
 
 ### Publish/push sub-projects
 
-- Useful for monorepos (projects with multiple sub-projects/packages): `yalc publish package-dir will perform publish operation in nested`package` folder of current working dir.
+- Useful for monorepos (projects with multiple sub-projects/packages): `yalc publish package-dir will perform publish operation in nested `package` folder of current working dir.
```
Comment (aleclarson) [inline README.md]: This line is already fixed by #57, so you can undo this change.

Comment (ndresx) [reply README.md]: Have reverted the change!

**Comment by aleclarson** [comment]
@whitecolor Friendly ping! :) Any idea when you'll get to review some of the open PRs? Much appreciated! 😊
